component/Form.jsx

```diff
@@ -0,0 +1,128 @@
+import styled from 'styled-components'
+import { useState, useEffect } from 'react'
+
+const FormBox = styled.form`
+  background-color: #eeeeee;
+  border-radius: 16px;
+  box-shadow: 0 8px 20px rgba(0, 0, 0, 0.08);
+  width: 450px;
+  padding: 32px;
+  height: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  gap: 12px;
+`
+
+const FormText = styled.label`
+  font-size: 1.2rem;
+  font-weight: 600;
+  color: #333;
+`
+
+const FormInput = styled.input`
+  font-size: 1.2rem;
+  font-weight: 600;
+  border: 0px;
+  color: #333;
+`
+
+const FormButton = styled.button`
+  font-size: 1rem;
+  font-weight: 600;
+  color: #333;
+`
+
+export default function Form({ mutateCards, onSubmit }) {
```

Comment:
```suggestion
export default function Form({ onSubmit }) {
```

Please remove `mutateCards` as you are not using it.